server/src/util.js

```diff
@@ -0,0 +1,65 @@
+const path = require('path');
+const YAML = require('yamljs');
+
+/**
+ * Fetches the input file extension
+ * @param {object} file - The input file received as form-data
+ * @returns {string} The extension of the input file
+ */
+function getFileExtension(file) {
```

Comment:
@pcgamer1 you don't need the documentation unless a tool is parsing it. Write readable code instead.
You could easily remove the documentation on top and call the method - `getInputFileExtension`
and let the param be `inputFile` and return string be `inputFileExtension`.

https://blog.usejournal.com/stop-writing-code-comments-28fef5272752 - Writing comments is that good a practice, use comments wisely

Reply:
Can you remove these unnecessary ones, better the naming for the rest of the methods here too? @pcgamer1